SimPEG/Regularization.py

```diff
@@ -1193,6 +1207,10 @@ def R(self, f_m):
             return self.stashedR
 
         eps = self.epsilon
+
+        if eps is None:
+            eps = 1e-8
```

Comment:
rather than setting a default here, can we do it on the property above? eg line 1178 (that way it is also documented).